### DISCUSSION TITLE
Massive Patch

### DISCUSSION THREAD

**Discussion by DannybotDev** [FunnyDannyG/FNFMegaMayhem#6] [Announcements]
Patch Notes

- Added Singularity
- Finished all difficulties of every chart except for overclocked (and megabyte?)
- Overhauled This One's Final Hours
- Removed Herobrine

<hr /><em>This discussion was created from the release <a href='https://github.com/DannybotDev/FNFMegaMayhem/releases/tag/v1.0.4-prealpha'>Massive Patch</a>.</em>

**Comment by FlashlightET** [comment]
this isnt minecraft. herobrine was never in the game
